pkg/apis/airflow/v1alpha1/airflow.go

```diff
@@ -167,9 +167,9 @@ func (r *AirflowCluster) getAirflowEnv(saName string, base *AirflowBase) []corev
 	dagFolder := AirflowDagsBase
 	if sp.DAGs != nil {
 		if sp.DAGs.Git != nil {
-			dagFolder = AirflowDagsBase + "/" + GitSyncDestDir + "/" + sp.DAGs.DagSubdir
+			dagFolder = AirflowDagsBase + GitSyncDestDir + "/" + sp.DAGs.DagSubdir
```

Comment:
Thanks. 
Just to be consistent with GitSyncDestDir, it may be better to remove the "/" from the const AirflowDagsBase instead.